File: app/src/main/java/art/luaj/hyperisle/ui/activity/MainActivity.java
package art.luaj.hyperisle.ui.activity;

import android.content.Context;
import android.os.Bundle;

import androidx.navigation.NavController;
import androidx.navigation.fragment.NavHostFragment;
import androidx.navigation.ui.NavigationUI;

import com.google.android.material.bottomnavigation.BottomNavigationView;

import art.luaj.hyperisle.R;
import art.luaj.hyperisle.databinding.ActivityMainBinding;
import art.luaj.hyperisle.ext.BaseActivity;
import art.luaj.hyperisle.ext.Tools;
import art.luaj.hyperisle.util.StatusBarUtil;

public class MainActivity extends BaseActivity {
    private Context context;
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        try {
            init();
            this.context = this;
            this.binding = ActivityMainBinding.inflate(getLayoutInflater());
            setContentView(binding.getRoot());

            NavHostFragment navFragment = (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.nav_fragment);
            NavController navController = navFragment.getNavController();
            BottomNavigationView navView = binding.navView;
            navView.setLabelVisibilityMode(BottomNavigationView.LABEL_VISIBILITY_SELECTED);

            NavigationUI.setupWithNavController(navView, navController);
            Tools.showAlert(this, "你好", true);
        } catch (Exception e) {
            Tools.showAlert(this, e.toString(), false);
        }
    }

    private void init() {
        StatusBarUtil.with(this).setTransparentStatusBar().setTransparentNavigationBar();
    }
}

File: app/src/main/java/art/luaj/hyperisle/ui/activity/MainActivity.kt
package art.luaj.hyperisle.ui.activity

import android.content.SharedPreferences
import android.os.Bundle
import art.luaj.hyperisle.databinding.ActivityMainBinding
import art.luaj.hyperisle.ui.base.BaseActivity

class MainActivity : BaseActivity() {
    private lateinit var binding: ActivityMainBinding
    private var sp: SharedPreferences? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        //setContentView(binding.root)
//        sp = getSharedPreferences(Config.APP_DEFAULT_STORAGE, MODE_PRIVATE)
//        if (sp!!.getBoolean("app_init", true)) {
//            startActivity(Intent(this, StartActivity::class.java))
//        }
    }
}